torchaudio/functional.py

```diff
@@ -946,7 +957,7 @@ def gain(waveform, gain_db=1.0):
     r"""Apply amplification or attenuation to the whole waveform.
 
     Args:
-       waveform (torch.Tensor): Tensor of audio of dimension (channel, time).
+       waveform (torch.Tensor): Tensor of audio of dimension (..., time).
```

Comment:
Maybe ```([...], channel, time)``` to indicate that we still follow that convention of channel x time?

Reply:
Duplicated comment.